spec/features/tags/user_sees_tags_spec.rb

```diff
@@ -1,24 +1,25 @@
 require 'rails_helper'
   RSpec.describe "User visits specific job page" do
     scenario "sees tags associated with that job" do
-      tag = Tag.create(title: "Computer")
       company = Company.create(name: "Xcel Energy")
       job = Job.create(title: 'Software Engineer', level_of_interest: 100, city: "Denver", company_id: company.id)
+      job.tags.create(title: "Computer")
```

Comment:
Great job finding the need to add the association between job and tags this way...